src/generator/gen_unicode.ml.inc

```diff
@@ -0,0 +1,135 @@
+
```

Comment:
A comment explaining the content of this file would be appropriate. It does not have to be long.